presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClusteringPredicatesExtractor.java

```diff
@@ -87,75 +63,139 @@ public CassandraClusteringPredicatesExtractor(List<CassandraColumnHandle> cluste
         return TupleDomain.withColumnDomains(notPushedDown);
     }
 
-    private static ClusteringPushDownResult getClusteringKeysSet(List<CassandraColumnHandle> clusteringColumns, TupleDomain<ColumnHandle> predicates)
+    private static ClusteringPushDownResult getClusteringKeysSet(List<CassandraColumnHandle> clusteringColumns, TupleDomain<ColumnHandle> predicates, String cassandraVersion)
     {
         ImmutableMap.Builder<ColumnHandle, Domain> domainsBuilder = ImmutableMap.builder();
-        ImmutableList.Builder<Set<Object>> clusteringColumnValues = ImmutableList.builder();
+        ImmutableList.Builder<String> clusteringColumnSql = ImmutableList.builder();
+        int lastclusteringColumn = 0;
```

Comment:
I think `currentClusterColumn` is more suitable name for this variable, since that's what you meant?